P0009/P0009.md

```diff
@@ -1355,28 +1357,37 @@ of rank equal to `sizeof...(Extents)`.
 
 [2]{.pnum} `extents<Extents...>` is a trivially copyable type.
 
-[3]{.pnum} $E_r$ is a _dynamic extent_ if it is equal to `dynamic_extent`, otherwise $E_r$ is a _static extent_.
+[3]{.pnum} Let $E_r$ be the $r^{th}$ element of $Extents$.
+
+[4]{.pnum} $E_r$ is a _dynamic extent_ if it is equal to `dynamic_extent`, otherwise $E_r$ is a _static extent_.
 For each $E_r$ equivalent to `dynamic_extent`,  
```

Comment:
```suggestion
For each $E_r$ that is a dynamic extent,
```
1. It's not clear why we need to say "equivalent" here.
2. It would be nicer to use the definition on line 1362 immediately after using it.

Reply:
done